doc/reference/reference_sql/sql_user_guide.rst

```diff
@@ -969,26 +993,33 @@ But comparisons of values of different types are allowed if the definition is SC
     in the same way as the :ref:`number <index-box_number>` type in
     NoSQL Tarantool.
 
+    Starting with Tarantool 2.10.1, these conversions which used to be legal are now illegal: |br|
+    Explicit cast from numeric to BOOLEAN, |br|
+    Explicit cast from BOOLEAN to numeric, |br|
+    Implicit cast from NUMBER to other numeric types for arithmetic or built-in functions. |br|
+    Implicit cast from numeric to STRING. |br|
+    Implicit cast from STRING to numeric.
+
 Examples of casts, illustrating the situations in the chart:
 
-``CAST(TRUE AS INTEGER)`` is legal because the intersection of the  "From BOOLEAN" row with the "To number"
+``CAST(TRUE AS STRING)`` is legal because the intersection of the  "From BOOLEAN" row with the "To STRING"
 column is ``A--`` and the first letter of ``A--`` is for explicit cast and A means Always Allowed.
-The result is 1.
+The result is 'TRUE'.
 
 ``UPDATE ... SET varbinary_column = 'A'`` is illegal because the intersection of the "From STRING" row with the "To VARBINARY"
 column is ``A--`` and the second letter of ``A--`` is for implicit cast (assignment) and - means not allowed.
 The result is an error message.
 
-``1.7E-1 > 0`` is legal because the intersection of the "From number" row with the "To number"
+``1.7E-1 > 0`` is legal because the intersection of the "From nueric" row with the "To numeric"
```

Comment:
I think here should be `numeric` instead of `nueric`.

Reply:
Oops. Yes.